src/Planet/planet-images/earth.svg

```diff
@@ -0,0 +1,89 @@
+	c1.007,0.665,2.141,1.118,3.336,1.314l41.344,6.894l-4.035,12.072l-21.542,7.192c-1.928,0.64-3.566,1.954-4.616,3.694
+	l-24.503,40.832l-39.356,23.616l-58.015,8.284c-4.206,0.597-7.329,4.189-7.337,8.438v25.595c0,2.261,0.896,4.436,2.5,6.032
+	l14.563,14.563V223l-35.492-23.667L37.019,160.71C61.513,104.743,106.407,60.208,162.57,36.166z M138.281,315.892l-38.563-7.721
+	l-14.333-28.59v-20.049l31.789-31.789l11.765,23.539c1.442,2.892,4.402,4.718,7.636,4.718h54.892l23.112,38.52
+	c1.544,2.568,4.317,4.138,7.312,4.138h23.718l-5.972,29.92l-32.301,32.301c-1.604,1.595-2.508,3.771-2.508,6.032v21.329
+	l-30.714,23.035c-2.15,1.612-3.413,4.138-3.413,6.825v30.816l-10.613-3.515l-14.982-37.471v-83.678
+	C145.106,320.201,142.256,316.703,138.281,315.892z M27.549,186.245l7.09,21.261c0.597,1.8,1.775,3.344,3.353,4.402l42.53,28.351
+	l-9.7,9.709c-1.604,1.595-2.5,3.771-2.5,6.032v25.595c0,1.322,0.307,2.628,0.896,3.814l17.063,34.127
+	c1.177,2.355,3.387,4.035,5.972,4.547l35.833,7.158v78.329c0,1.083,0.205,2.158,0.606,3.165l17.063,42.658
+	c0.93,2.329,2.85,4.129,5.23,4.923l25.595,8.532c0.853,0.29,1.749,0.444,2.653,0.444c4.709,0,8.532-3.822,8.532-8.532v-38.392
+	l30.714-23.035c2.15-1.612,3.413-4.138,3.413-6.825v-22.063l31.627-31.627c1.186-1.194,1.996-2.705,2.329-4.36l8.532-42.658
+	c0.921-4.624-2.073-9.112-6.697-10.042c-0.546-0.111-1.109-0.162-1.664-0.162H226.72l-23.112-38.52
+	c-1.544-2.568-4.317-4.138-7.312-4.138h-54.449l-14.657-29.409c-1.229-2.457-3.558-4.172-6.271-4.607
+	c-2.705-0.469-5.477,0.427-7.397,2.389l-11.074,11.031V204.81c0-2.261-0.896-4.436-2.5-6.032l-14.563-14.563v-14.666l52.401-7.482
+	c1.126-0.162,2.21-0.546,3.182-1.135l42.658-25.595c1.194-0.717,2.201-1.723,2.918-2.918l23.974-39.954l22.6-7.542
+	c2.551-0.836,4.556-2.841,5.392-5.392l8.532-25.595c1.476-4.471-0.947-9.299-5.426-10.775c-0.41-0.137-0.836-0.239-1.263-0.316
+	l-49.381-8.233l-8.677-5.784c53.69-17.473,111.841-15.314,164.08,6.1l-15.587,7.781h-32.113c-3.251-0.017-6.228,1.809-7.678,4.718
+	l-17.063,34.127c-2.099,4.215-0.392,9.342,3.831,11.441c0.358,0.179,0.734,0.333,1.109,0.461l25.595,8.532
+	c1.536,0.52,3.191,0.589,4.761,0.188l31.721-7.926l23.24,7.738c0.333,0.119,0.674,0.213,1.024,0.273l42.658,8.532
+	c2.218,0.452,4.53,0,6.407-1.263l19.418-12.951c9.555,10.647,18.147,22.123,25.663,34.289l-41.976,13.992l-45.448-7.576
+	l-15.016-22.523c-1.578-2.363-4.215-3.78-7.056-3.797h-51.19c-2.261,0-4.436,0.896-6.032,2.5l-25.595,25.595
+	c-1.903,1.903-2.798,4.581-2.414,7.243l8.532,59.721c0.427,2.952,2.372,5.469,5.119,6.629l59.721,25.595
+	c1.715,0.742,3.617,0.896,5.426,0.435l30.466-7.619l18.625,12.439v29.562c-0.017,1.322,0.282,2.628,0.853,3.814l33.273,66.453
+	v32.113c0,2.261,0.896,4.436,2.5,6.032l17.848,17.857c-79.156,105.511-228.852,126.874-334.363,47.717
+	C32.19,386.696-1.826,282.422,27.549,186.245z"/>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+</svg>

```

Comment:
Noting for our future selves that this is a placeholder that can be safely deleted later.